static/usage/v7/toast/position-anchor/react.md

```diff
@@ -0,0 +1,41 @@
+```tsx
+import React from 'react';
+import { IonButton, IonContent, IonFooter, IonHeader, IonTitle, IonToast, IonToolbar } from '@ionic/react';
+
+function Example() {
+  return (
+    <IonHeader id="header">
```

Comment:
I think all the stuff in this `return` needs to be wrapped by something, even if it's just a `<></>`

Reply:
Whoops, good catch 👀